[Radar] Add Radar api docs (#6577)
cloudflare/cloudflare-docs: content/radar/investigate/netflows.md

{{<Aside type="note" header="Netflows products">}}
`HTTP` only includes web traffic to our zones, while `ALL` also includes traffic to all other services, like [Spectrum](https://www.cloudflare.com/en-gb/products/cloudflare-spectrum/), [Magic Transit](https://www.cloudflare.com/en-gb/magic-transit/), [1.1.1.1](https://1.1.1.1/dns/), our public DNS resolver, and others.
 {{</Aside>}}

Let's look at both in two [autonomous systems](https://www.cloudflare.com/en-gb/learning/network-layer/what-is-an-autonomous-system/).

First, we'll look at [AS3243](https://radar.cloudflare.com/as3243), a portuguese local Internet Service Provider or ISP (parameters for all traffic: `name=AS3243_all&product=ALL&dateRange=1d&asn=3243` and for just HTTP traffic: `name=AS3243_http&product=HTTP&dateRange=1d&asn=3243`):

```bash
curl -X GET "https://api.cloudflare.com/client/v4/radar/netflows/timeseries?name=meo_all&product=ALL&dateRange=1d&asn=3243&name=meo_http&product=HTTP&dateRange=1d&asn=3243&format=json&aggInterval=1h" \
     -H "Authorization: Bearer <API_TOKEN>"
```

> And here's the abbreviated response:

Reviewer: ```suggestion
This is the abbreviated response:
```

```json
"AS3243_all": {
	"timestamps": ["2022-11-08T14:00:00Z", "2022-11-08T15:00:00Z", "..."],
	"values": ["0.565885", "0.586434", "..."]
},
"AS3243_http": {
	"timestamps": ["2022-11-08T14:00:00Z", "2022-11-08T15:00:00Z", "..."],
	"values": ["0.548564", "0.568329", "..."]
},
```

> We see that `HTTP` traffic values are similar to `ALL` traffic values - this means that most traffic Cloudflare sees as coming from this AS, is traffic to websites served by Cloudflare's [CDN](https://www.cloudflare.com/en-gb/learning/cdn/what-is-a-cdn/) product.

Reviewer: ```suggestion
`HTTP` traffic values are similar to `ALL` traffic values. This means that most traffic Cloudflare receives from this AS is traffic to websites served by Cloudflare's [CDN](https://www.cloudflare.com/en-gb/learning/cdn/what-is-a-cdn/) product.
```

```bash
curl -X GET "https://api.cloudflare.com/client/v4/radar/netflows/timeseries?name=AS174_all&product=ALL&dateRange=1d&asn=174&name=AS174_http&product=HTTP&dateRange=1d&asn=174&name=AS3243_all&product=ALL&dateRange=1d&asn=3243&name=AS3243_http&product=HTTP&dateRange=1d&asn=3243&format=json&aggInterval=1h" \
     -H "Authorization: Bearer <API_TOKEN>"
```

which would lead to a response like this:

```json
"AS174_all": {
  "timestamps": ["2022-11-08T14:00:00Z", "2022-11-08T15:00:00Z", "..."],
  "values": ["0.917348", "1.0", "..."]
},
"AS174_http": {
  "timestamps": ["2022-11-08T14:00:00Z", "2022-11-08T15:00:00Z", "..."],
  "values": ["0.381777", "0.408091", "..."]
},
"AS3243_all": {
  "timestamps": ["2022-11-08T14:00:00Z", "2022-11-08T15:00:00Z", "..."],
  "values": ["0.317136", "0.328652", "..."]
},
"AS3243_http": {
  "timestamps": ["2022-11-08T14:00:00Z", "2022-11-08T15:00:00Z", "..."],
  "values": ["0.307429", "0.318505", "..."]
}
```

> Cloudflare sees more traffic from AS174 than it sees from AS3243.

Reviewer: ```suggestion
This response shows how Cloudflare receives more traffic from AS174 than from AS3243.
```

Reviewer: Trying to be inclusive and not use the word "see". But I'm not sure if "receive" is the correct word here.
